samples/javascript_nodejs/52.logger-bot/README.md

```diff
@@ -0,0 +1,69 @@
+# loggerBot
+This bot demonstrates the transcript logging middleware in the nodejs SDK. The files `CustomLogger.d.ts` and `CustomLogger.js` contain a custom implementation of the logging middleware.
+
+This bot has been created using [Microsoft Bot Framework][10], it shows how to create a simple echo bot with state. The bot maintains a simple counter that increases with each message from the user. This bot example uses [`restify`][1].
+
+# To run the bot
+- Install modules and start the bot
+    ```bash
+    npm i & npm start
+    ```
+    Alternatively you can also use nodemon via
+    ```bash
+    npm i & npm run watch
+    ```
+
+# Testing the bot using Bot Framework Emulator
+[Microsoft Bot Framework Emulator][2] is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+- Install the Bot Framework emulator from [here][3]
+
+## Connect to bot using Bot Framework Emulator **V4**
+- Launch Bot Framework Emulator
+- File -> Open Bot Configuration
+- Select `loggerBot.bot` file
+
```

Comment:
V4 Emulator can load and render transcript files. You should a) test this and make sure emulator can render the transcripts and b) add this to the readme. 

The 'resouces' tab in emulator automatically shows your .chat files found under your .bot file path and .transcript files found under /transcripts folder under the same parent folder as the .bot file. 

In that spirit, you might also want to consider setting logFilePath to instead be transcriptsPath=./transcripts in your .env file.

Reply:
I've changed the filetype to `.transcript` for the moment, but if you have some time I think it might be good to go over making the logs compatible with the emulator by default. The first way I did it isn't compatible off the bat. However it's doable, but it might be a little complicated for the first logging sample bot, and might include some bad logging practices if I do it the way I can  currently think of.